FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Angel Aguilar on 7/14/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedImagesMapper {
```

Comment:
`FeedImageMapper` is defined as a class, so we can instantiate it as `FeedImageMapper()`. However, all its methods are static - it has no instance methods or properties.

It may be confusing for other developers to be able to instantiate a class but then not be able to do anything with it.

As you'll see in the following lectures, you can make the initializer private in such cases since the `FeedImageMapper` is just a namespace for static methods, with no instance-specific behavior.

Or you can make it an enum with no cases, which can never be instantiated!

Both solutions will disallow instantiation at compile time so that `FeedImageMapper()` won't compile ✅

Reply:
The convention in this codebase is to omit the default access level keyword of `internal`, please remove it.
```suggestion
final class FeedImagesMapper {
```